spec/unit/provider/package/zypper_spec.rb

```diff
@@ -154,14 +145,14 @@ def shell_out_expectation!(command, options=nil)
         "zypper --non-interactive --no-gpg-checks install "+
         "--auto-agree-with-licenses emacs=1.0"
       )
-      provider.upgrade_package("emacs", "1.0")
+      provider.upgrade_package(["emacs"], ["1.0"])
     end
     it "should run zypper upgrade without gpg checks" do
```

Comment:
on tests with the Chef::Config variable set to false, the other tests the default.